qemu/tests/getfd.py

```diff
@@ -52,7 +52,7 @@ def has_fd(pid, filepath):
         path = os.path.join(fdfiles_dir, name)
         fd = os.open(path, os.O_RDWR | os.O_CREAT)
         response = vm.monitor.getfd(fd, name)
-        os.close(fd)
+        vm.monitor.closefd(fd, name)
```

Comment:
Thanks @luckyh @PyLearner 

Code has been updated, could you help review it again?

Thanks.